sdk/api/sdk.api

```diff
@@ -488,11 +488,6 @@ public abstract interface class com/mapbox/maps/renderer/OnFpsChangedListener {
 	public abstract fun onFpsChanged (D)V
 }
 
-public final class com/mapbox/maps/renderer/RenderHandlerThread$sam$i$java_lang_Runnable$0 : java/lang/Runnable {
-	public fun <init> (Lkotlin/jvm/functions/Function0;)V
-	public final synthetic fun run ()V
-}
-
```

Comment:
pretty curious how did this happen

Reply:
since its internal class changes, we can skip these as discussed offline.